test.js

```diff
@@ -131,4 +133,31 @@ describe('anymatch', function() {
       matchers.pop();
     });
   });
+
+  describe('windows paths', function() {
+    anymatch.__set__('platform', 'win32');
+
```

Comment:
Actually the problem I'm going to have with that is that I don't expose micromatch's options in anymatch and just rely on the defaults. I don't think I want to expand the API to do this, definitely not just for the sake of the tests.

Maybe something internal/undocumented.